builder/image/slugbuilder/builder/install-buildpacks

```diff
@@ -8,19 +8,22 @@ download_buildpack() {
     buildpack_name=$(basename $buildpack_url)
     buildpack_commit="$2"
 
+    echo "Fetching $buildpack_name..."
+
     set +e
-    git clone --branch $buildpack_commit --depth 1 $buildpack_url $BUILDPACK_INSTALL_PATH/$buildpack_name
+    git clone --branch $buildpack_commit --depth 1 $buildpack_url $BUILDPACK_INSTALL_PATH/$buildpack_name &>/dev/null
```

Comment:
Can you redirect all streams in one fell swoop like that? I've always done `>/dev/null 2>&1`.